third_party/kythe/java/com/google/devtools/kythe/extractors/shared/CAMPFIRE

```diff
@@ -18,5 +18,21 @@
         "//third_party/protobuf:protobuf_jar"
       ]
     }
+  },
+  {
+    "name": "file_vnames",
```

Comment:
Although it is true that this is not used in Shipshape, I would rather bring in entire packages from Kythe instead of editing CAMPFIRE files to remove portions of a package. If you think this is sufficiently different from the other files in this package, perhaps it should be a different package kythe-side.

Note that I would expect other developers that want to import portions of Kythe to do the same thing (copy over specific directories).